### DISCUSSION TITLE
想问一下 deprecatedKeys 字段是移除了吗？

### DISCUSSION THREAD

**Discussion by aoguai** [gkd-kit/gkd#520] [Q&A] [answered]
我在 docs 项目和 gkd 本体项目中没找到相关代码了已经，但是在 [gkd-kit/subscription](https://github.com/gkd-kit/subscription) 规则项目中还有规则存在这一字段。

还是说这个字段仅在 规则项目 中有用？

**Comment by lisonge** [comment, marked as answer]
> 还是说这个字段仅在 规则项目 中有用？

是的
